slices/erlang-ftp.yaml

```diff
@@ -0,0 +1,17 @@
+package: erlang-ftp
+
+essential:
+  - erlang-ftp_copyright
+
+slices:
+  libs:
+    essential:
+      - erlang-base_bins
```

Comment:
why do these libs need the erland-base_bins?

NOTE: this is a recurrent comment and there's already a related thread below. so I'll skip similar comments for other SDFs and let's address the problem in the ongoing thread from Rafid

Reply:
I am going to change _libs slices to _modules slices for those beam files. As for the dependency question, it is explained in the [first comment](https://github.com/canonical/chisel-releases/pull/258#discussion_r1802944113).

Reply:
Let's continue this in https://github.com/canonical/chisel-releases/pull/258#discussion_r1802799227.

Why do you say beams are modules? Aren't they binary encoded? If so, they are much closer to an executable than to a module. Given their special type, BEAM, why not simply call the slice, `beams`?